### ISSUE TITLE
Can you add CPU temperature

### ISSUE THREAD

**Issue by greekstreet** [PlugaruT/wingpanel-indicator-sys-monitor#29]
It would be nice if CPU temperature can be displayed on the panel. No need to type "sensors" in terminal to check it.

**Comment by PlugaruT** [comment]
Yes, it would be nice to have that but I have to figure it out how to read the data...
Thanks for the suggestion.

**Comment by umbynos** [comment]
It would be really nice to have it
